browser/src/VariantList/variantTableColumns.tsx

```diff
@@ -45,8 +46,22 @@ const getConsequenceDescription = (contextType: any) => {
       return ' for consequence in this transcript'
   }
 }
-
-const variantTableColumns = [
+export type VariantTableColumn = {
```

Comment:
Nice! I know some of these larger types can be a bit of a chore, particularly depending on your familiarity with the code base, but I do think it both helps to build understanding of the codebase, and gives all the juicy benefits of static typing :)